README.md

```diff
@@ -53,4 +53,41 @@ An overview of this method is discussed in the [full documentation](https://mncr
 This code contains a Julia implementation of the MATLAB code included as supplementary material with [Crowe & Johnson 2024](https://doi.org/10.1017/jfm.2024.619) and also includes a version of the previously unrealeased SQG version discussed in [Johnson & Crowe 2023](https://doi.org/10.1017/jfm.2023.87) and [Crowe & Johnson 2023](https://doi.org/10.1017/jfm.2023.607).
 For those interested in the original (layered QG only) implementation, it can be found [here](https://github.com/mncrowe/QGDipoles.m). 
 
- 
+## Contributing
+
+Issue should be submitted [here](https://github.com/mncrowe/QGDipoles.jl/issues/) and discussions or questions [here](https://github.com/mncrowe/QGDipoles.jl/discussions).
```

Comment:
```suggestion
Issues should be submitted [here](https://github.com/mncrowe/QGDipoles.jl/issues/) and discussions or questions [here](https://github.com/mncrowe/QGDipoles.jl/discussions).
```
----
Note that you don't _have to_ use GitHub discussions if you don't find that feature helpful (I don't, personally). It's totally fine to direct discussions/questions to GitHub issues too. Whichever you prefer!